vm/src/stdlib/winapi.rs

```diff
@@ -189,28 +189,30 @@ fn _winapi_CreateProcess(
         .as_mut()
         .map_or_else(null_mut, |w| w.as_mut_ptr());
 
-    let mut procinfo = unsafe { std::mem::zeroed() };
-    let ret = unsafe {
-        processthreadsapi::CreateProcessW(
-            app_name,
-            command_line,
-            null_mut(),
-            null_mut(),
-            args.inherit_handles,
-            args.creation_flags
-                | winbase::EXTENDED_STARTUPINFO_PRESENT
-                | winbase::CREATE_UNICODE_ENVIRONMENT,
-            env as _,
-            current_dir,
-            &mut si as *mut winbase::STARTUPINFOEXW as _,
-            &mut procinfo,
-        )
+    let procinfo = unsafe {
+        let mut procinfo = std::mem::MaybeUninit::uninit();
+        let ret = unsafe {
```

Comment:
Ping, warning came up which you probably didn't see since you aren't on windows.